pkg/upgrade/upgrader/helper.go

```diff
@@ -86,3 +86,13 @@ func isOperatorUpgraded(componentName string, namespace string,
 	}
 	return nil
 }
+
+// Remove the suffix only if it is present
+// at the end of the string
+func removeSuffixFromEnd(str, suffix string) string {
```

Comment:
to be on the safer side, can we call this function only if the URI is something like:`openebs/cstor-pool-amd64:` 

this can avoid cases where users might have copied the images to custom repositories with a suffix like -amd64.

Reply:
Done